components/builder-api/habitat/config/config.toml

```diff
@@ -14,11 +14,11 @@ app_private_key = "{{pkg.svc_files_path}}/builder-github-app.pem"
 [oauth]
 {{toToml cfg.oauth}}
 
-{{~#eachAlive bind.router.members as |member|}}
+{{~#each bind.router.members as |member|}}
```

Comment:
Was this intentional?

Reply:
It was. @christophermaier added a feature that made using `eachAlive` unnecessary.